pkg/mac/common.sh

```diff
@@ -19,7 +19,10 @@
 
 set -e -x
 
-export ARCH=${ARCH:-arm64}
+if [ ! -n "$ARCH" ]; then
```

Comment:
```suggestion
if [ -z "$ARCH" ]; then
```

Prefer positive condition to double negative condition.